style.css

```diff
@@ -1614,4 +1623,14 @@ input#Email {
     width: 200px;
     height: 40px;
   }
+
+  .lineD {
+    margin-top: 698px;
+    width: 100%;
+  }
+
+  .subInput :nth-child(4) {
+    display: flex;
+    justify-content: flex-start;
+  }
```

Comment:
- [ ] Great work🎉😍 thus far on incorporating the style changes mentioned by the [previous reviewer.](https://github.com/bhushan354/My-Portfolio/pull/4/files#r1263689373)
- [ ]  However, I feel there is still an opportunity for further improvement in terms of the overall project styling. 
- [ ] I have prepared a video for your reference, where I point out specific areas that could benefit from modifications in order to align your design with the Figma design provided.
- [ ]  By ensuring your design matches the Figma design, you'll be able to captivate the attention of future recruiters. 
- [ ] Thank you for your attention to this matter.🤔🎉🥱

https://github.com/bhushan354/My-Portfolio/assets/118761303/88051611-4140-4019-aacd-5a377c08994f